homeassistant/components/device_tracker/googlehome.py

```diff
@@ -77,8 +77,8 @@ def __init__(self, hass, config):
     async def async_update_info(self):
         """Ensure the information from Google Home is up to date."""
         _LOGGER.debug('Checking Devices...')
-        await self.scanner.scan_for_devices()
         await self.scanner.get_scan_result()
+        await self.scanner.scan_for_devices()
```

Comment:
How can you first get the result and then scan for devices?

Reply:
There are already devices there, from the last triggered scan.

But the unit does not update that list unless you tell it to.
The result of that scan can take anywhere between 5-10 sec to finish, and there is no way to tell when it's finished, this will ensure that the flow continues to run.

Reply:
So you update the list with the result from the previous scan?

Reply:
My response was a little uninformative...
The scan continues to run for 60sec, but have will have useful information after about 5-10 sec.
So it will get the devices that were initialized by the previous run, but the data are up to date with each run.

The unit is slow, so if I grab the data right after triggering the scan it returns no devices `[]`

Reply:
scratch that, I see that I have caped it at 5sec.
Will see if there is another way,